ci-operator/step-registry/upi/vsphere/windows/post/upi-vsphere-windows-post-commands.sh

```diff
@@ -23,3 +23,12 @@ do
         echo "$(date -u --rfc-3339=seconds) - $(rm -vf "${f}")"
     fi
 done
+
+# remove any created folders. Each line in the created_folders.txt file will contain the name of a folder
+folder_file="${SHARED_DIR}/created_folders.txt"
+while read -r line; do
+    echo "$(date -u --rfc-3339=seconds) - destroying folder ${line}..."
```

Comment:
```suggestion
    echo "$(date -u --rfc-3339=seconds) - removing folder ${line}"
```

Reply:
I'd like to stick to destroying as that is the language used by the govc command.